4-layout/19-lines-and-blocks/index.html

```diff
@@ -38,10 +38,12 @@ <h2>Services</h2>
       <h2>Contact</h2>
       <p>Interested in upskilling with Rendezvous Tutoring? Feel free to sign-up with your e-mail below to learn more!</p>
       <form>
-        <input type="email" placeholder="e-mail here" required />
-        <input type="submit" value="Sign Up" />
+      <label for="email">Email:</label>
+<input type="email" id="email" placeholder="e-mail here" required />
+        <button type="submit"> Sign up</button>
```

Comment:
Let’s keep this as an `<input>` for now to match what was taught in the HTML course and avoid confusion, even though `<button>` elements might be a better choice overall.

Reply:
Ok.